### PR TITLE
Add bomb damage indicator

### PR DESCRIPTION
**Pull request by mkot2**
Pasted from UC
Fully accurate on maps with standard bomb ranges

### CODE REVIEW

**Comment by wemanzoz** [comment]
based

**Comment by mkot2** [comment]
> 
> 
> based

wdym

**Comment by YeetingWaterbottle** [comment]
Nice work, I will use this

**Comment by wemanzoz** [comment]
> Nice work, I will use this

thats what i meant @mkot2

**Comment by mkot2** [comment]
> 
> 
> > Nice work, I will use this
> 
> thats what i meant @mkot2

ah ok sorry

File: Osiris/Hacks/Misc.cpp [hunk 1]
```diff
@@ -383,6 +386,60 @@ void Misc::drawBombTimer() noexcept
 
     ImGui::textUnformattedCentered(ss.str().c_str());
 
+    //if (localPlayer) {
+    float finalBombDamage = 0.f;
+
+    for (auto i = interfaces->engine->getMaxClients(); i <= interfaces->entityList->getHighestEntityIndex(); i++)
```
Comment (blackMasterLord) [inline Osiris/Hacks/Misc.cpp:392]: this is for idiots, ye?🤦🏻‍♂️
https://github.com/danielkrupinski/Osiris/blob/2135abad64fde6f45e14bdbee49cf0fbe24c029f/Osiris/GameData.cpp#L261-L264

Comment (mkot2) [reply Osiris/Hacks/Misc.cpp:392]: > 
> 
> this is for idiots, ye?🤦🏻‍♂️
> 
> https://github.com/danielkrupinski/Osiris/blob/2135abad64fde6f45e14bdbee49cf0fbe24c029f/Osiris/GameData.cpp#L261-L264

how to get the entity???

Comment (blackMasterLord) [reply Osiris/Hacks/Misc.cpp:392]: just store bomb handle
```
void BombData::update() noexcept
{
    if (memory->plantedC4s->size > 0 && (!*memory->gameRules || (*memory->gameRules)->mapHasBombTarget())) {
        if (const auto bomb = (*memory->plantedC4s)[0]; bomb && bomb->c4Ticking()) {
            blowTime = bomb->c4BlowTime();
            timerLength = bomb->c4TimerLength();
            defuserHandle = bomb->c4Defuser();
            handle = bomb->handle();
            if (defuserHandle != -1) {
                defuseCountDown = bomb->c4DefuseCountDown();
                defuseLength = bomb->c4DefuseLength();
            }

            if (*memory->playerResource) {
                const auto& bombOrigin = bomb->origin();
                bombsite = bombOrigin.distTo((*memory->playerResource)->bombsiteCenterA()) > bombOrigin.distTo((*memory->playerResource)->bombsiteCenterB());
            }
            return;
        }
    }
    blowTime = 0.0f;
}
```

**Comment by Frouk3** [comment]
There's command that allows you to know range of C4

**Comment by mkot2** [comment]
> 
> 
> There's command that allows you to know range of C4
> Cvar: map_showbombradius

its not a cvar, if you know how to get the values then tell me

**Comment by Shuvi-Moro** [comment]
https://i.imgur.com/BkPXWOX.jpeg
Doesn't work in the menu (gives an error)

**Pull request closed without merging**